lib/ImageProcessor.js

```diff
@@ -58,25 +59,27 @@ class ImageProcessor {
         if ( config.exists("backup") ) {
             const backup = config.get("backup");
             backup.acl = backup.acl || acl;
+            backup.cacheControl = (backup.cacheControl !== undefined) ? backup.cacheControl : cacheControl;
```

Comment:
`backup.cacheControl || cacheControl` would be better to omit `null` or other falsy value as same as above line.

Reply:
This is to allow the specific configuration to *remove* cacheControl (by setting to `null`). So the global value can set it to "x", and a specific resize can set it back to `null`. Also setting to `null` prevents using the original file's cache control.

Reply:
Ah, that make sense. the `null` value will remove cacheControl.